module-cheatsheets/scRNA-seq-advanced-cheatsheet.md

```diff
@@ -219,8 +221,25 @@ Read the [`AUCell` package documentation](https://bioconductor.org/packages/rele
 |-------------------------------|--------------------------------------------------------------|--------------------------------|--------------------------------------------------------------------------|
 | `GSEABase` | [`GeneSet()`](https://rdrr.io/bioc/GSEABase/man/GeneSet-class.html) | Gene set | Constructs a gene set as a `GeneSet` object for use with `AUCell` |
 | `GSEABase` | [`GeneSetCollection()`](https://rdrr.io/bioc/GSEABase/man/GeneSetCollection-class.html) | Gene set collection | Constructs a collection of gene sets as a `GeneSetCollection` object for use with `AUCell`
-| `AUCell` | [`AUCell_buildRankings()`](https://rdrr.io/bioc/AUCell/man/AUCell_buildRankings.html) | Build cell rankings | Builds a ranking of genes for each cell that is used to calculate the recovery curve | 
+| `AUCell` | [`AUCell_buildRankings()`](https://rdrr.io/bioc/AUCell/man/AUCell_buildRankings.html) | Build cell rankings | Builds a ranking of genes for each cell that is used to calculate the recovery curve |
 | `AUCell` | [`AUCell_calcAUC()`](https://rdrr.io/bioc/AUCell/man/AUCell_calcAUC.html) | Calculate AUC | Calculates the area under the recovery curve (AUC) for each gene set in each cell |
 | `AUCell` | [`AUCell_exploreThresholds()`](https://rdrr.io/bioc/AUCell/man/AUCell_exploreThresholds.html) | Explore thresholds | Calculates thresholds in AUC values that can be used to assign cells; optionally makes assignments and produces histograms |
 
 
+
+## `bluster`
+
+Read the [`bluster` package documentation](https://rdrr.io/bioc/bluster/) and vignettes on its usage:
+
+* [Flexible clustering for Bioconductor](https://rdrr.io/bioc/bluster/f/vignettes/clusterRows.Rmd)
+* [Assorted clustering diagnostics](https://rdrr.io/bioc/bluster/f/vignettes/diagnostics.Rmd)
+
+
+| Library/Package      | Piece of Code      | What it's called    | What it does  |
+|----------------------|--------------------|---------------------|---------------|
+| `bluster`| [`clusterRows()`](https://rdrr.io/bioc/bluster/man/clusterRows.html)| Cluster rows of a matrix | Perform clustering using a variety of algorithms on a matrix-like object |
+| `bluster`| [`KGraphParam()`](https://rdrr.io/bioc/bluster/man/KmeansParam-class.html)| K-means clustering | Run k-means clustering using `kmeans()` when provided to `bluster::clusterRows()` |
```

Comment:
Clustering function name fix, as well as a clarification that this function only sets up parameters. 
```suggestion
| `bluster`| [`KmeansParam()`](https://rdrr.io/bioc/bluster/man/KmeansParam-class.html)| K-means clustering parameters | Set up parameters to run clustering using `kmeans()` within `bluster::clusterRows()` |
```

Reply:
I agree!!! I originally had written this with "set up parameters" as you suggested, but the bluster docs didn't include this language so I removed it. But I think we're right.